File: Assets/VirtualCity/ProtoDefine/Java/mall/ReqConfirmReceiptMessage.java
package com.kingston.jforgame.server.game.mall.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.entity.mall.Comment;
import com.kingston.jforgame.server.game.entity.mall.Order;
import com.kingston.jforgame.server.game.mall.MallDataPool;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

@MessageMeta(module = Modules.SHOPPING,cmd = MallDataPool.REQ_CONFIRM)
public class ReqConfirmReceiptMessage extends Message {
    /**评论信息*/
    @Protobuf(order = 1,fieldType = FieldType.OBJECT)
    private Order order;

    public ReqConfirmReceiptMessage() {
    }

    public ReqConfirmReceiptMessage(Order order) {
        this.order = order;
    }

    public Order getOrder() {
        return order;
    }

    public void setOrder(Order order) {
        this.order = order;
    }
}

File: Assets/VirtualCity/ProtoDefine/ReqConfirmReceiptMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class ReqConfirmReceiptMessage {
    /**评论信息*/
[ProtoMember(1)]
    public Order order;

    public ReqConfirmReceiptMessage() {
    }

    public ReqConfirmReceiptMessage(Order order) {
        this.order = order;
    }

    public Order getOrder() {
        return order;
    }

    public void setOrder(Order order) {
        this.order = order;
    }
}
}